Method/book.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../item'
+require 'json'
+require 'date'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state, :title, :color, :labels, :genre
+
+  def initialize(publish_date, title, publisher, color, cover_state)
+    super(publish_date)
+    @title = title
+    @publisher = publisher
+    @color = color
+    @cover_state = cover_state
```

Comment:
`cover_state` should be a string, however, it is being saved as a boolean. Kindly fix this issue. 
![image](https://github.com/Booze33/Ruby-Capstone/assets/33268927/cbadae2b-cf5d-4e90-a6d5-1bee18c7ef7e)
![image](https://github.com/Booze33/Ruby-Capstone/assets/33268927/d9da2027-9ca2-4bce-b2f7-76e8a1ee350e)

Reply:
Hello @muneebulrehman 

Thank you for your feedback. The changes have been made.

Regards